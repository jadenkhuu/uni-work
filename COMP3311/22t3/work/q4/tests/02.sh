psql racing -c "select * from q4('Eddie') order by horse;"
