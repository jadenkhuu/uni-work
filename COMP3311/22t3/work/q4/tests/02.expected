 horse | average 
-------+---------
 Eddie |   22142
(1 row)
